src/main/java/io/wisetime/connector/jira/JiraDao.java

```diff
@@ -176,7 +180,9 @@ void createWorklog(final Worklog worklog) {
         .namedParam("timeSpent", worklog.getTimeWorked())
         .namedParam(
             "createdDate",
-            ZonedDateTime.of(worklog.getCreated(), getJiraDefaultTimeZone()).format(dateTimeFormatter)
```

Comment:
On the old logic, we just assigned a timezone but it did not convert the time to the specified timezone